seadssite/static/js/graph.js

```diff
@@ -257,15 +288,57 @@ function fetch_bar_graph(url) {
     request.send();
 }
 
+function generate_gauge(data) {
+    var length = data.data.length;
+    var gauge = c3.generate({
+        padding: {
+                top: 0,
+                right: 0,
+                bottom: 0,
+                left: 0,
+            },
+        bindto: '#gauge',
+        data: {
+            columns: [
+                ['data'].concat(Math.round((data.data[length-1].energy*360) * 1000) / 1000)
+            ],
+            type: 'gauge',
+        },
+        gauge: {
+           label: {
+               format: function(value, ratio) {
+                   return value;
+               },
+               show: true // to turn off the min/max labels.
+           },
+           min: 0, // 0 is default, //can handle negative min e.g. vacuum / voltage / current flow / rate of change
+           max: 10, // 100 is default
+           units: ' kW',
+           width: 50 // for adjusting arc thickness
+        },
+        color: {
+            pattern: ['60B044', '#F6C600', '#F97600', '#FF0000'], // the three color levels for the percentage values.
```

Comment:
can you make these into constants so its not just values, like GREEN, YELLOW, RED or something? just for readablility